compiler/src/fInfo.ml

```diff
@@ -24,19 +24,22 @@ let f_annot_empty = {
     f_user_annot          = [];
   }
 
-type subroutine_info = { returned_params : int option list }
+type arg_ret_info = { returned_params : int option list }
+(** When a non-inlined function returns a reg ptr, it has to be one of its
+    arguments. [returned_params] associates to each return value the index of
+    the corresponding argument if it is a reg ptr, and [None] otherwise. *)
```

Comment:
```suggestion
(** When a non-inlined function returns a `ptr` array, it has to be one of its
    arguments. [returned_params] associates to each return value the index of
    the corresponding argument if it is a `ptr` array, and [None] otherwise. *)
```

Reply:
Idem.